<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Box</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Serial Number</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Quantity:</string>
     </property>
    </widget>
   </item>
<!--   <item row="4" column="0">-->
<!--    <widget class="QLabel" name="label_3">-->
<!--     <property name="text">-->
<!--      <string>Email: </string>-->
<!--     </property>-->
<!--    </widget>-->
<!--   </item>-->
<!--   <item row="5" column="0" colspan="2">-->
<!--    <widget class="QLineEdit" name="lineEditEmail"/>-->
<!--   </item>-->

   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="LineSerialNum"/>
   </item>
<!--   <item row="3" column="0" colspan="2">-->
<!--    <widget class="QLineEdit" name="LineQuantity"/>-->
<!--   </item>-->
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
